snap/snapcraft.yaml

```diff
@@ -626,6 +626,10 @@ parts:
           install -DT "./cmd/$service/$service" "$SNAPCRAFT_PART_INSTALL/bin/$service"
 
           case "$service" in
+          "core-metadata")
+              install -DT "./cmd/core-metadata/res/configuration.toml" "$SNAPCRAFT_PART_INSTALL/config/core-metadata/res/configuration.toml"
```

Comment:
No needed as this is done at the bottom of the `for/case` for all the services

Reply:
Isn't the bottom one acts as default case if no match is found ?

Reply:
Ahh yep, because you added this case that will not be hit.

Reply:
This is good as it copies also `uom.toml` to the snap package (readonly, [`SNAP`](https://snapcraft.io/docs/environment-variables) env var). But we also need to add code to copy it over to the snap's writable area ([`SNAP_DATA`](https://snapcraft.io/docs/environment-variables) env var) on install time. The given path `UOM_UOMFILE: $SNAP_DATA/config/core-metadata/res/uom.toml` expects it to be under the writable area.

Currently only `configuration.toml` files get copied:
https://github.com/edgexfoundry/edgex-go/blob/e3ca38a5709b6fd134cd3d7a24fe413a337b80c3/snap/local/hooks/cmd/install/install.go#L118-L143

This is really an unnecessary complication in how the snap is packaged. Maybe we should refactor and simply copy the whole `res` directory.

For now, I'll add a commit to copy `uom.toml` from `SNAP` to `SNAP_DATA`.